components/_expanding-components/uncheck-radio-button.js

```diff
@@ -0,0 +1,37 @@
+export const UncheckRadioButton = function(elementClass) {
+  const expandingComponent = elementClass
+  let expandingElement
+
+  const cacheDom = function() {
+    expandingElement = document.querySelectorAll('.' + expandingComponent)
+  }
+
+  const bindEvents = function() {
+    expandingElement.forEach(labelElement => {
+      labelElement.addEventListener('click', () => findInputForLabel(labelElement))
+    })
+  }
+
+  const findInputForLabel = function(labelElement) {
+    const labelElementFor = labelElement.htmlFor
+    const inputId = labelElementFor
+
+    if (labelElementFor) {
```

Comment:
if you go with above you should just check for inputId

Reply:
Done